src/components/BookCard.js

```diff
@@ -1,45 +1,43 @@
 import React from 'react';
-// eslint-disable-next-line import/no-extraneous-dependencies
-import Proptypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-const BookCard = ({ book }) => {
-  const renderBooks = book.map((book) => (
-    <>
-      <div className="bookContainer">
-        <div className="bookTitle">
-          <p>{ book.category }</p>
-          <p>{ book.title }</p>
-          <p>{ book.author }</p>
-          <ul className="buttonsList">
-            <li><button type="button">Comments</button></li>
-            <li><button type="button">Remove</button></li>
-            <li><button type="button">Edit</button></li>
-          </ul>
-        </div>
-        <div className="bookChart">
-          <div>Chart</div>
-          <div>
-            <p>{ book.progres }</p>
-            <p>COMPLETED</p>
-          </div>
-        </div>
-        <div className="bookChapter">
-          <p>{ book.currentChapter }</p>
-          <p>{ book.chapterNumber }</p>
-          <button type="button">Update Progress</button>
+const BookCard = () => {
+  const books = useSelector((store) => store.books.books);
+  const dispatch = useDispatch();
+
+  const renderBooks = books.map((book) => (
+    <div key={book.item_id} className="bookContainer">
+      <div className="bookTitle">
+        <p>{book.category}</p>
+        <p>{book.title}</p>
+        <p>{book.author}</p>
+        <ul className="buttonsList">
+          <li><button type="button">Comments</button></li>
+          <li><button type="button" onClick={() => dispatch(removeBook(book.item_id))}>Remove</button></li>
+          <li><button type="button">Edit</button></li>
+        </ul>
+      </div>
+      <div className="bookChart">
+        <div>Chart</div>
+        <div>
+          <p>progres</p>
+          <p>COMPLETED</p>
```

Comment:
- [ ] According to the requirement, though it is not mentioned directly, the BookCard or Book is supposed to be a child of BooksList meaning BookCard and BookList should have a parent-child-relationship.

Books.js or BookCard.js
```
const Book = (props) => {
  const { book } = props;
  const { item_id, title, author, category } = book;

  return (
    <li className="book flex-row">
      <div className="bookInfo">
        <div className="flex-column">
          <span className="bookCategory">{category}</span>
          <span className="bookTitle">{title}</span>
          <span className="bookAuthor">{author}</span>
        </div>
      </div>
    </li>
  );
};
```

Then Books.js or BookList.js
will be something like
```
import BookCard.js

const Books = () => {

//dummy data
const books = [
{
author: 'you',
name: 'bookname'
}
]

  return (
    <main>
      <ul className="booksContainer flex-column">
        {books.map((book) => (<Book key={book.item_id} book={book} />))}
      </ul>
    </main>
  );
};
```